astropy/io/fits/tests/test_image.py

```diff
@@ -168,6 +168,53 @@ def test_open_3(self):
         finally:
             fits.conf.lazy_load_hdus = True
 
+    def test_fortran_array(self):
+        # Test that files are being correctly written+read for "C" and "F" order arrays
+        dims = (3,7)
+        size = np.product(dims)
+        a = np.zeros(dims, order='C')
+        b = np.zeros(dims, order='F')
+        a[:] = np.arange(size).reshape(*dims)
+        b[:] = np.arange(size).reshape(*dims)
+
+        # writting to str specified files
+        fits.PrimaryHDU(data=a).writeto(self.temp('a_str.fits'))
```

Comment:
`self.temp` incurs method call overhead. Since this appears to be called repeatedly below for both `a` and `b`, I recommend setting it once per array and store them in variables for re-use.